src/accelerate/hooks.py

```diff
@@ -202,6 +202,12 @@ def remove_hook_from_module(module: nn.Module, recurse=False):
             module.forward = module._old_forward
         delattr(module, "_old_forward")
 
+    # Remove accelerate added warning hooks
+    module.__dict__.pop('to', None)
+    module.__dict__.pop('cuda', None)
+    module.__dict__.pop('xpu', None)
+    module.__dict__.pop('npu', None)
```

Comment:
Proposal: what if we saved these all as `_accelerate_added_attributes` and then this way we can just pop the lot of them without needing to worry about keeping everything aligned in the future

Reply:
I agree. This also makes it easier to find where these attributes come from, as a reader can search for `_accelerate_added_attributes` and quickly find the origin.

Reply:
Make sense !